internal/controller/services/monitoring/monitoring_controller_actions.go

```diff
@@ -161,13 +167,15 @@ func deployMonitoringStackWithQuerier(ctx context.Context, rr *odhtypes.Reconcil
 	// Early exit if no metrics configuration
 	if monitoring.Spec.Metrics == nil {
 		setConditionFalse(rr, status.ConditionMonitoringStackAvailable, status.MetricsNotConfiguredReason, status.MetricsNotConfiguredMessage)
+		setConditionFalse(rr, status.ConditionNamespaceRestrictedMetricsAvailable, status.MetricsNotConfiguredReason, status.MetricsNotConfiguredMessage)
```

Comment:
Does it make sense then to show this condition? It provides little added value if it is the only way to have these metrics - i.e. namespaced.